mindsdb/api/http/namespaces/util.py

```diff
@@ -19,6 +20,20 @@ def get(self):
         '''Checks server avaliable'''
         return {'status': 'ok'}
 
+
+@ns_conf.route('/ping_native')
+class PingNative(Resource):
+    @ns_conf.doc('get_ping_native')
+    def get(self):
+        ''' Checks server use native for learn or analyse.
+            Will return right result only on Linux.
+        '''
+        for p in psutil.process_iter(['name']):
+            if p.info['name'] == 'mindsdb_native_process':
+                return {'native_process': True}
+        return {'native_process': False}
```

Comment:
overall (given that we also implement the "predict guard" this might be sufficient to be able to tell wether a mindsdb instance is working or not, with some caveates (i.e. maybe we keep pinging this for ~5 minutes before closing) :)